x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/model_select_option.tsx

```diff
@@ -142,11 +143,41 @@ export const ModelMenuPopover: React.FC<{
   );
 };
 
+export interface LicenseBadgeProps {
+  licenseType: string;
+  licensePageUrl?: string;
+}
+
+export const LicenseBadge: React.FC<LicenseBadgeProps> = ({ licenseType, licensePageUrl }) => {
+  const licenseLabel = i18n.translate(
+    'xpack.enterpriseSearch.content.indices.pipelines.modelSelectOption.licenseBadge.label',
+    {
+      defaultMessage: 'License: {licenseType}',
+      values: {
+        licenseType,
+      },
+    }
+  );
+
+  return (
+    <EuiBadge color="hollow">
+      {licensePageUrl ? (
+        <EuiLink target="_blank" href={licensePageUrl}>
+          {licenseLabel}
+        </EuiLink>
+      ) : (
+        <p>{licenseLabel}</p>
```

Comment:
Why did you add this `<p>`? I'm not sure I've see this before, I have had to wrap `<EuiBadge>` with a `<span>` to ensure they dont grow in certain scenarios and wondering if this `<p>` solves that instead.